src/app/modals/recoverwallet/recoverwallet.component.html

```diff
@@ -6,13 +6,104 @@
+    </div>
+    <div class="col-md-2">
+      <input [(ngModel)]="words[7]" type="text" name="recovery" [disabled]="isDisabled" class="form-control recovery-word">
+    </div>
+    <div class="col-md-2">
+      <input [(ngModel)]="words[8]" type="text" name="recovery" [disabled]="isDisabled" class="form-control recovery-word">
+    </div>
+    <div class="col-md-2">
+      <input [(ngModel)]="words[9]" type="text" name="recovery" [disabled]="isDisabled" class="form-control recovery-word">
+    </div>
+    <div class="col-md-2">
+      <input [(ngModel)]="words[10]" type="text" name="recovery" [disabled]="isDisabled" class="form-control recovery-word">
+    </div>
+        <div class="col-md-2">
+      <input [(ngModel)]="words[11]" type="text" name="recovery" [disabled]="isDisabled" class="form-control recovery-word">
+    </div>
+    <div class="col-md-2">
+      <input [(ngModel)]="words[12]" type="text" name="recovery" [disabled]="isDisabled" class="form-control recovery-word">
+    </div>
+    <div class="col-md-2">
+      <input [(ngModel)]="words[13]" type="text" name="recovery" [disabled]="isDisabled" class="form-control recovery-word">
+    </div>
+    <div class="col-md-2">
+      <input [(ngModel)]="words[14]" type="text" name="recovery" [disabled]="isDisabled" class="form-control recovery-word">
+    </div>
+    <div class="col-md-2">
+      <input [(ngModel)]="words[15]" type="text" name="recovery" [disabled]="isDisabled" class="form-control recovery-word">
+    </div>
+    <div class="col-md-2">
+      <input [(ngModel)]="words[16]" type="text" name="recovery" [disabled]="isDisabled" class="form-control recovery-word">
+    </div>
+    <div class="col-md-2">
+      <input [(ngModel)]="words[17]" type="text" name="recovery" [disabled]="isDisabled" class="form-control recovery-word">
+    </div>
+    <div class="col-md-2">
+      <input [(ngModel)]="words[18]" type="text" name="recovery" [disabled]="isDisabled" class="form-control recovery-word">
+    </div>
+    <div class="col-md-2">
+      <input [(ngModel)]="words[19]" type="text" name="recovery" [disabled]="isDisabled" class="form-control recovery-word">
+    </div>
+    <div class="col-md-2">
+      <input [(ngModel)]="words[20]" type="text" name="recovery" [disabled]="isDisabled" class="form-control recovery-word">
+    </div>
+    <div class="col-md-2">
+      <input [(ngModel)]="words[21]" type="text" name="recovery" [disabled]="isDisabled" class="form-control recovery-word">
+    </div>
+    <div class="col-md-2">
+      <input [(ngModel)]="words[22]" type="text" name="recovery" [disabled]="isDisabled" class="form-control recovery-word">
+    </div>
+    <div class="col-md-2">
+      <input [(ngModel)]="words[23]" type="text" name="recovery" [disabled]="isDisabled" class="form-control recovery-word">
+    </div>
+  </div>
+</div>
+
+<hr style="margin-top: 40px;">
+
+<!--
```

Comment:
I purposefully left it out for the time being until I got to work on the firsttime dialog, where it's re-used.. I'm now creating that, which will also be used here.